commit e91bd7465816f474617dcb4bbfe72f3594c5783c
Author: Adhemerval Zanella <adhemerval.zanella@linaro.com>
Date:   Thu Mar 31 10:51:51 2016 -0300

    Fix tst-dlsym-error build
    
    This patch fixes the new test tst-dlsym-error build on aarch64
    (and possible other architectures as well) due missing strchrnul
    definition.
    
    	* elf/tst-dlsym-error.c: Include <string.h> for strchrnul.

diff --git a/elf/tst-dlsym-error.c b/elf/tst-dlsym-error.c
index 11b0358..fb084c5 100644
--- a/elf/tst-dlsym-error.c
+++ b/elf/tst-dlsym-error.c
@@ -20,6 +20,7 @@
 #include <gnu/lib-names.h>
 #include <stdio.h>
 #include <stdlib.h>
+#include <string.h>
 
 /* Used to disambiguate symbol names.  */
 static int counter;
